projects/1.20/assets/ruins/philipsruins/lang/zh_cn.json

```
@@ -0,0 +1,40 @@
+{
+    "block.philipsruins.red_sand_stone_brick": "红砂岩砖",
+    "block.philipsruins.sand_stone_brick": "砂岩砖",
+    "block.philipsruins.red_sand_stone_tiles": "红砂岩瓷砖",
+    "block.philipsruins.obsidian_brick": "黑曜石砖",
+    "block.philipsruins.soul_sand_bricks": "灵魂沙砖",
+    "block.philipsruins.sandstone_tiles": "砂岩瓷砖",
+    "block.philipsruins.dark_brick": "暗色石砖",
+    "block.philipsruins.mossy_dark_brick": "苔藓暗色石砖",
+    "block.philipsruins.bone_bricks": "骨头砖",
+    "block.philipsruins.bone_bricks_slab": "骨头砖台阶",
+    "block.philipsruins.bone_chunk_block": "骨头块",
+    "block.philipsruins.chiseled_bone_bricks": "錾制骨头砖",

```

```suggestion
    "block.philipsruins.chiseled_bone_bricks": "雕纹骨头砖",
```
此改动在1.19.3进行，请注意参照对应版本的原版翻译